benchmarking-tests/benchmark_utils.py

```diff
@@ -36,6 +44,38 @@ def run_set_statsd_host(args):
         json.dump(conf, f)
 
 
+def run_gather_metrics(args):
+    # setup a rethinkdb connection
+    conn = r.connect(args.bigchaindb_host, 28015, 'bigchain')
+
+    # setup csv writer
+    csv_file = open(args.csvfile, 'w')
+    csv_writer = csv.writer(csv_file)
+
+    # query for the number of transactions on the backlog
+    num_transactions = r.table('backlog').count().run(conn)
+    logger.info('Starting gathering metrics. {} transasctions in the backlog'.format(num_transactions))
+
+    # listen to the changefeed
+    try:
+        for change in r.table('bigchain').changes().run(conn):
+            # check only for new blocks
+            if change['old_val'] is None:
+                block_num_transactions = len(change['new_val']['block']['transactions'])
+                time_now = time.time()
+                logger.info('{} {}'.format(time_now, block_num_transactions))
```

Comment:
@vrde Maybe something should be added to [our Python Style Guide](https://github.com/bigchaindb/bigchaindb/blob/master/PYTHON_STYLE_GUIDE.md#using-the--operator-or-format-to-format-strings)? Right now, it says to use format() always.